### ISSUE TITLE
Dropdown button in pickers demo doesn't have a sufficient tap target size for accessibility.

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#21578]
~~This might effect all dropdown buttons~~ All that needs to be done is ensure that semantics are merged properly, since it appears that there _should_ be enough room in the button to support this.

Otherwise we will need to add exterior padding with materialTapTargetSize flag.

EDIT:
The button in question is a custom implementation.
